library/core/class.controller.php

```diff
@@ -874,52 +874,56 @@ public function finalize() {
     /**
      *
      *
-     * @param string $AssetName
+     * @param string $assetName
      */
-    public function getAsset($AssetName) {
-        if (!array_key_exists($AssetName, $this->Assets)) {
+    public function getAsset($assetName) {
+        if (!array_key_exists($assetName, $this->Assets)) {
             return '';
         }
-        if (!is_array($this->Assets[$AssetName])) {
-            return $this->Assets[$AssetName];
+        if (!is_array($this->Assets[$assetName])) {
+            return $this->Assets[$assetName];
         }
 
         // Include the module sort
-        $Modules = Gdn::config('Modules', array());
+        $modules = array_change_key_case(c('Modules', []));
+        $sortContainer = strtolower($this->ModuleSortContainer);
+        $applicationName = strtolower($this->Application);
+
         if ($this->ModuleSortContainer === false) {
-            $ModuleSort = false; // no sort wanted
-        } elseif (array_key_exists($this->ModuleSortContainer, $Modules) && array_key_exists($AssetName, $Modules[$this->ModuleSortContainer]))
-            $ModuleSort = $Modules[$this->ModuleSortContainer][$AssetName]; // explicit sort
-        elseif (array_key_exists($this->Application, $Modules) && array_key_exists($AssetName, $Modules[$this->Application]))
-            $ModuleSort = $Modules[$this->Application][$AssetName]; // application default sort
+            $moduleSort = false; // no sort wanted
+        } elseif (isset($modules[$sortContainer][$assetName])) {
+            $moduleSort = $modules[$sortContainer][$assetName]; // explicit sort
+        } elseif (isset($modules[$applicationName][$assetName])) {
+            $moduleSort = $modules[$applicationName][$assetName]; // application default sort
+        }
```

Comment:
All of the real code changes are in lines 888 through 898. The rest is just variable renames.